components/molecules/GhOpenGraphImg/gh-open-graph-img.tsx

```diff
@@ -0,0 +1,19 @@
+import { generateGhOgImage } from "lib/utils/github";
+import Image from "next/image";
+import InvalidImage from "img/404 Image.svg";
+import { useEffect, useState } from "react";
+
+interface GhOpenGraphImgProps {
+  githubLink: string;
+}
+
+const GhOpenGraphImg = ({ githubLink }: GhOpenGraphImgProps): JSX.Element => {
+  const { isValid, url } = generateGhOgImage(githubLink);
+
+  return (
+    // eslint-disable-next-line @next/next/no-img-element
```

Comment:
please stop disabling eslint rules and state your problems

Reply:
It's a nextjs warning to not use the native `img` tag in next https://nextjs.org/docs/messages/no-img-element

Reply:
That seems aggressive for linter, but disabling it seems like a band-aid. Are there other alternatives @OgDev-01?

Reply:
Another alternative is to disable globally in the `.eslintrc` rules
<img width="450" alt="image" src="https://user-images.githubusercontent.com/62995161/220386470-f62c5ed4-54c7-41ab-beff-7a6cbaedbef1.png">